[MRG] Adding release for 0.2.

adam2392 opened the pull request:
PR Description
--------------

Release for 0.2.

Merge checklist
---------------

Maintainer, please confirm the following before merging:

- [ ] All comments resolved
- [ ] This is not your own PR
- [ ] All CIs are happy
- [ ] PR title starts with [MRG]
- [ ] [whats_new.rst](https://github.com/mne-tools/mne-hfo/blob/master/doc/whats_new.rst) is updated
- [ ] PR description includes phrase "closes <#issue-number>"

adam2392 commented:
@larsoner Hi! I am trying to make a new release now following: https://github.com/mne-tools/mne-bids/wiki/How-to-make-a-release#prepare-stable-docs

I'm at the step of pushing a release to pypi. It specifies I need admin access. Is this something I need for my pypi account, or should I just use my personal one `adam2392`?

larsoner commented:
You are listed as the only maintainer on https://pypi.org/project/mne-hfo/, so it seems like it should work.

If you set up 2-factor authentication the instructions are different. For example I have entries like this in my `~/.pypirc`:
```
[mne]
  repository = https://upload.pypi.org/legacy/
  username = __token__
  password = <redacted>
```
where the `<redacted>` password was a token generated on PyPi as a one-time-shown token generated in the "API tokens" section of my PyPi profile that allows me to upload to MNE. And when I upload MNE I do:
```
twine upload dist/* --repository mne --verbose
```

I do not appear to have admin permissions on `mne-hfo` so I can't tweak anything there. Usually you want at least two people to have access to settings and such, feel free to add me (Eric89GXL) as an admin/whatever on PyPi.

adam2392 commented:
Oh okay! If I can do everything using my own account, then that's great. I just updated and did the official v0.2 release now which brings things fully under the mne umbrella. 

I also added you as a maintainer on pypi since that seemed easy enough. Thanks for the help!
